src/npctrade.cpp

```diff
@@ -328,7 +328,17 @@ TAB key to switch lists, letters to pick items, Enter to finalize, Esc to quit,\
                 draw_border( w_tmp );
                 wrefresh( w_tmp );
                 // TODO: use input context
-                help = inp_mngr.get_input_event().get_first_input() - 'a';
+                help = inp_mngr.get_input_event().get_first_input();
+                if( help >= 'a' && help <= 'z' ) {
+                    help -= 'a';
+                }
+                else if( help >= 'A' && help <= 'Z' ) {
```

Comment:
Move `else if` clause one line up, so it should look like `} else if(...`. The same with `else` below.